SocialNetworkApi/Domain/Comment.cs

```diff
@@ -0,0 +1,17 @@
+namespace SocialNetwork.Domain
+{
+    public class Comment
+    {
+        public Guid Id { get; set; }
+        public Guid PostId { get; set; }
+        public Guid UserId { get; set; }
+        public string Content { get; set; }
+        public DateTime CreatedAt { get; set; }
+
+        public Comment()
```

Comment:
Be careful for warnings:
Non-nullable property 'Content' must contain a non-null value when exiting constructor. Consider adding the 'required' modifier or declaring the property as nullable.